ruff.toml

```diff
@@ -73,6 +73,7 @@ ignore = [
 # Suppress nuisance warnings about module-import-not-at-top-of-file (E402) due to workaround for #4476
 "setuptools/__init__.py" = ["E402"]
 "pkg_resources/__init__.py" = ["E402", "ANN204"]
+"pkg_resources/tests/test_resources.py" = ["PT031"]
```

Comment:
Given `pkg_resources` is planned for removal, I'm not even gonna bother checking what this triggered on ^^